source/common/stream_info/filter_state_impl.cc

```diff
@@ -31,5 +31,29 @@ const FilterState::Object* FilterStateImpl::getDataGeneric(absl::string_view dat
   return it->second.get();
 }
 
+void FilterStateImpl::addToListGeneric(absl::string_view data_name,
+                                       std::unique_ptr<Object>&& data) {
+  // TODO(Google): Remove string conversion when fixed internally.
+  const std::string name(data_name);
+  if (list_storage_.find(name) == list_storage_.end()) {
+    list_storage_[name] = std::vector<std::unique_ptr<FilterState::Object>>();
+  }
+
+  // absl::string_view will not convert to std::string without an explicit case; see
+  // https://github.com/abseil/abseil-cpp/blob/master/absl/strings/string_view.h#L328
```

Comment:
nit, suggestion: Should this be above the declaration of name?  It's not clear in isolation what this comment refers to.